### ISSUE TITLE
[CRAN Release]: <v0.6.6> 1st Dec

### ISSUE THREAD

**Issue by shajoezhu** [insightsengineering/rtables#745]
### PRs

- [ ] PR 1

### Issues

- [ ] Issue 1

### Pre-requisites

- [ ] Make sure that high priority bugs (label "priority" + "bug") have been resolved before going into the release.
- [ ] Review old/hanging PRs before going into the release (Optional).
- [ ] Revisit R-package's lifecycle badges (Optional).
- [ ] Make sure that all upstream dependencies of this package that need to be submitted to CRAN were accepted before going into release activities.
- [ ] Make sure integration tests are green 2-3 days before the release. Look carefully through logs (check for warnings and notes).
- [ ] Decide what gets merged in before starting release activities.

### Release Checklist

- [ ] Update NEWS.md file: make sure it reflects a holistic summary of what has changed in the package.
- [ ] Remove the additional fields (`Remotes` and `Config/Needs/*`) from the DESCRIPTION file where applicable.
- [ ] Increase versioned dependency on {package name} to >=X.X.X (Optional).
- [ ] Make sure that the minimum dependency versions are updated in the DESCRIPTION file for the package and its reverse dependencies (Optional).
- [ ] Create a pull request to make necessary bug fixes/changes (add "[skip vbump]" in the pr title), and after merging the PR, tag the update(s) as a release candidate v < intended release version > -rc < release candidate iteration > on the main branch.
- [ ] Build the package locally using the command:`R CMD build .` which will generate a .tar.gz file necessary for the CRAN submission.
- [ ] Submit the package that was build in the previous step via this form: https://cran.r-project.org/submit.html.
- [ ] Address CRAN feedback, tag the package vX.X.X-rc(n+1) and repeat the submission to CRAN whenever necessary.
- [ ] Get the package accepted and published on CRAN.
- [ ] If the additional fields were removed, add them back in a separate PR, and then merge the PR back to main. Note: Take precautionary measures to ensure that the version bump does not take place on a merge.
- [ ] Create a git tag with the final version set to X.X.X on the main branch.

### Testing

- [ ] Integration tests results - accepted.
- [ ] UAT results - accepted.
- [ ] All testing activities are finalized.

### Release Feedback

- [ ] Fix 1
- [ ] Enhancement 1
- [ ] Defect 1

### Post-release Checklist

- [ ] Make sure that the package is published to internal repositories.
- [ ] Review and update installation instructions for the package wherever needed (Optional).
- [ ] Update all integration tests to reference the new release.
- [ ] Ensure a new dev version (.9XXX) is added to the NEWS.md file and DESCRIPTION file as a placeholder for release notes.
- [ ] Announce the release on ________.

### Decision tree

Click [here](https://github.com/insightsengineering/.github/blob/main/.github/ISSUE_TEMPLATE/RELEASE_DECISION_TREE.md) to see the release decision tree.
